addons/account/models/account_move.py

```diff
@@ -2208,7 +2208,7 @@ def copy(self, default=None):
         return copied_am
 
     def _sanitize_vals(self, vals):
-        if 'invoice_line_ids' in vals and 'line_ids' in vals:
+        if vals.get('invoice_line_ids', False) and vals.get('line_ids', False):
```

Comment:
The second parameter of `get` is actually useless in this case and makes the code redundant/less readable.
Please only provided it when necessary